optimum/exporters/onnx/__main__.py

```diff
@@ -405,6 +405,8 @@ def main_export(
 
         # Saving the model config and preprocessor as this is needed sometimes.
         model.config.save_pretrained(output)
+        if hasattr(model, "generation_config") and model.generation_config is not None:
+            model.generation_config.save_pretrained(output)
```

Comment:
Quick tip: I usually use `getattr` in this situation because I do both checks and access at the same time.
```suggestion
        generation_config = getattr(model, "generation_config")
        if generation_config is not None:
            generation_config.save_pretrained(output)
```